### ISSUE TITLE
Jetpack Plan purchase: better alternate flow when VaultPress can't be installed

### ISSUE THREAD

**Issue by jeherve** [Automattic/wp-calypso#9354]
#### Steps to reproduce
0. Start from a Jetpack site already connected to WordPress.com
1. Go to wordpress.com/plans
2. Select your site
3. Purchase the Premium plan
4. If you get an error when Calypso attempts to install VaultPress, this leaves me at a dead-end. We need some better “what next” link out from this view, even if it just goes back to My Plan:

<img width="388" alt="screen-shot-2016-11-11-at-4-06-58-pm" src="https://cloud.githubusercontent.com/assets/426388/20275075/60818a50-aa97-11e6-9b23-84a0a8e00aed.png">

p2JiWH-1tm-p2

**Comment by ryelle** [comment]
Here's a brain-dump on what's already here:

There are 2 intentional next-step actions you can take from this screen: In the main error message, you can click the link on "manually installing" to see instructions on how to proceed by yourself, or you can click the Contact Support button which should take you to the support form. I'm not sure how a link back to My Plan could be dropped in here, especially because there is no continued flow from there. If you go back to My Plan, you're basically giving up on installing these plugins.

We don't currently have a central page for service registration keys (though we've had an issue for it for a while, #205 ) so there's not a good place to direct users who might want to try later (except to their email, where they have the receipt with keys). If we made one - maybe on My Plan - it would make sense to direct users to this even if autoconfig fails. We already do this for success, "Continue" takes you to My Plan:

![screen shot 2016-11-14 at 1 03 25 pm](https://cloud.githubusercontent.com/assets/541093/20276618/00631750-aa6b-11e6-823c-d9e5e99beca2.png)

pinging @beaulebens since you reported this, and @rickybanister since we worked together on the flow/error messages.

**Comment by beaulebens** [comment]
Hrm, you're right about all the constraints and current links etc.

When I hit this, it felt like a total road-block/disaster. Maybe that was because there was so much red? Clicking either of those links felt like admitting defeat, and like it wasn't going to help me get my site set up right now.

Maybe for the manual installation instructions, we could just do something inline? So rather than a big red error with a link out to a massive wall of instructions, we tell them that we couldn't automatically configure their site, provide them with a link to download the plugin, their API key, and a quick one-liner about how to configure things?

FWIW, the specific error I got "you are not the owner of this site" is a really weird one, and it feels like if I got all the way through the process of setting things up and buying a plan, then I should be "the owner of the site", or at least be able to get VaultPress configured properly one way or another.
